src/test/resources/data/DvInstance1/doi-10-5072-fk2-u9gjxt-datacite.v1.0.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Be careful that any real data that you use in tests must be absolutely open. Just a reminder ;)

Reply:
I've used the open test data from IQSS